rosbag2_py/src/rosbag2_py/format_bag_metadata.cpp

```diff
@@ -147,6 +147,7 @@ std::string format_bag_meta_data(const rosbag2_storage::BagMetadata & metadata)
   info_stream << "Bag size:          " << format_file_size(
     metadata.bag_size) << std::endl;
   info_stream << "Storage id:        " << metadata.storage_identifier << std::endl;
+  info_stream << "ROS Distro:        " << metadata.ros_distro << std::endl;
```

Comment:
Need to change expectation in `InfoEndToEndTestFixture::info_end_to_end_test` accordingly to the changes.
Currently, the test fails with an error message
```
/tmp/ws/src/rosbag2/rosbag2_tests/test/rosbag2_tests/test_rosbag2_info_end_to_end.cpp:49
Value of: output
Expected: contains regular expression "\nFiles:             cdr_test_0.mcap\nBag size:          .*B\nStorage id:        mcap\nDuration:          0\\.151s\nStart:             Apr  .+ 2020 .*:.*:36.763 \\(1586406456\\.763\\)\nEnd:               Apr  .+ 2020 .*:.*:36.914 \\(1586406456\\.914\\)\nMessages:          7\nTopic information: "
  Actual: "\nFiles:             cdr_test_0.mcap\nBag size:          11.8 KiB\nStorage id:        mcap\nROS Distro:        \nDuration:          0.151s\nStart:             Apr  8 2020 20:27:36.763 (1586406456.763)\nEnd:               Apr  8 2020 20:27:36.914 (1586406456.914)\nMessages:          7\nTopic information: Topic: /array_topic | Type: test_msgs/msg/Arrays | Count: 4 | Serialization Format: cdr\n                   Topic: /test_topic | Type: test_msgs/msg/BasicTypes | Count: 3 | Serialization Format: cdr\n\n"
```

Reply:
Done